tests/test_calculator.hh

```diff
@@ -323,7 +323,8 @@ namespace rascal {
     ~SimpleStructureManagerNLCCStrictFixture() = default;
 
     const std::string filename{
-        "reference_data/CaCrP2O7_mvc-11955_symmetrized.json"};
+        "reference_data/CaCrP2O7_mvc-11955_symmetrized.json",
```

Comment:
wait, what is this doing? We don't have a `std::vector<std::string>` at the end, only a single `std::string` initialized with two `const char*`. And AFAIK, there is [no constructor for `std::string` taking two `const char*`](https://en.cppreference.com/w/cpp/string/basic_string/basic_string).

Is `"reference_data/small_molecule.json"` being converted to `size_type` here?

Reply:
I am not sure why it did not crash...